examples/cross-language-end-to-end-tests/common.exp

```diff
@@ -0,0 +1,42 @@
+#
```

Comment:
Please remove this line and could you check why the copyright header script did not catch this.